code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -1,4 +1,4 @@
-/mob/living/carbon/human/examine(mob/user) //User is the person being examined
+/mob/living/carbon/human/examine(mob/user)

```

Comment:
why did you remove the comment

Reply:
oh yeah I forgot to mention it, it's wrong/outdated

user is the examin-er, src the examin-ed